eos-install-google-chrome

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash -e
+
+# Downloads Google Chrome and unzips to the user's home directory,
+# with a desktop launcher
+
+exit_with_error() {
```

Comment:
As for the case with the Firefox script, it would be a good idea to make sure that ${TEMP_DIR} gets removed whenever this function is called. Sorry I did not notice this before, when writing my patches on top of yours.